Index: /@tapjs/run/dist/esm/index.js
===================================================================
--- /@tapjs/run/dist/esm/index.js
+++ /@tapjs/run/dist/esm/index.js
@@ -5,8 +5,9 @@
 // same process as the @tapjs/core Base class. node:repl loads
 // node:domain, which conflicts with async-hook-domains used by tap
 import { args, config, mainCommand } from './main-config.js';
 process.title = 'tap';
+void (async () => {
 switch (mainCommand) {
     case 'help':
         ;
         (await import('./help.js')).help(args, config);
@@ -52,5 +53,6 @@
         ;
         (await import('./list.js')).list(args, config);
         break;
 }
-//# sourceMappingURL=index.js.map
\ No newline at end of file
+})()
+//# sourceMappingURL=index.js.map
